executor/simple.go

```diff
@@ -1031,6 +1382,25 @@ func (e *SimpleExec) executeAlterUser(ctx context.Context, s *ast.AlterUserStmt)
 		}
 	}
 
+	sysSession, err := e.getSysSession()
+	defer clearSysSession(ctx, sysSession)
+	if err != nil {
+		return err
+	}
+	sqlExecutor := sysSession.(sqlexec.SQLExecutor)
+	// session isolation level changed to  READ-COMMITTED
+	_, err = sqlExecutor.ExecuteInternal(ctx, "set tx_isolation = 'READ-COMMITTED'")
+	if err != nil {
+		return err
+	}
+	if _, err := sqlExecutor.ExecuteInternal(ctx, "begin PESSIMISTIC"); err != nil {
```

Comment:
Please try to follow the same case style, say, "BEGIN PESSIMISTIC" or "begin pessimistic"

Reply:
done